consensus/parlia/parlia.go

```diff
@@ -1751,20 +1755,22 @@ func (p *Parlia) applyTransaction(
 	return nil
 }
 
-// GetJustifiedNumberAndHash returns the highest justified block's number and hash on the branch including and before `header`
-func (p *Parlia) GetJustifiedNumberAndHash(chain consensus.ChainHeaderReader, header *types.Header) (uint64, common.Hash, error) {
```

Comment:
If it is better to use a similar function signature as p.snapshot?
```go
func (p *Parlia) GetJustifiedNumberAndHash(chain consensus.ChainHeaderReader, header *types.Header, parents []*types.Header) (uint64, common.Hash, error) {
```

Reply:
p.snapshot defination is bad, we define it by following clique,
it's params `parents` has different meaning with all other interfaces, 
<img width="1235" alt="image" src="https://github.com/bnb-chain/bsc/assets/122502194/79d796d4-2783-44b1-975d-f969bcbab4c3">